docker/compose_files/docker-compose.yml

```diff
@@ -76,13 +86,41 @@ services:
 
   worker-manager-cpu:
     image: codalab/server:${CODALAB_VERSION}
-    command: cl-worker-manager --worker-manager-type ${CODALAB_WORKER_MANAGER_TYPE} --max-workers ${CODALAB_WORKER_MANAGER_MAX_CPU_WORKERS} --server ${CODALAB_SERVER} --queue ${CODALAB_WORKER_MANAGER_CPU_QUEUE} --search request_gpus=0
+    command: |
+      cl-worker-manager
+      --server ${CODALAB_SERVER}
+      --min-workers ${CODALAB_WORKER_MANAGER_MIN_CPU_WORKERS}
+      --max-workers ${CODALAB_WORKER_MANAGER_MAX_CPU_WORKERS}
+      --search request_gpus=0
+      --worker-tag ${CODALAB_WORKER_MANAGER_CPU_TAG}
+      --sleep-time ${CODALAB_WORKER_MANAGER_SECONDS_BETWEEN_WORKERS}
+      --worker-idle-seconds ${CODALAB_WORKER_MANAGER_IDLE_SECONDS}
+      ${CODALAB_WORKER_MANAGER_TYPE}
```

Comment:
Args that apply to all worker manager types need to go before the worker manager type and args that are specific to this worker manager type need to go after the argument. 

The problem with this setup is that if we ever add another worker manager type in the future, we'd need to add a separate entry in `docker-compose.yml` for it with the correct argument structure for it + we'd need to add all its arguments as arguments to `codalab_service.py` as well.

I don't know if there's an ideal solution here. Maybe making `--worker-manager-args` a single freeform string argument to `codalab_service` and passing just that in `docker-compose.yml` to the worker manager? Or getting rid of the subcommand structure for the `worker-manager` types and adding every argument for every type of worker manager into the same argument namespace and passing in everything that's set. 

I also feel like this decision can be  postponed to when a second worker manager type is about to be added.

Reply:
Can the order of the arguments be documented?

Reply:
I guess there's going to be Slurm and Azure Batch...yeah, it's a bit unfortunate that we're going to get the 2 x 3 explosion for CPU and GPU types.  Actually, maybe a better solution would be to have the worker manager read the environment variables directly, so that we can just have two entries here.

Reply:
I still think it might be better to decide this when the next set of worker managers are actually added and we can decide what's more ergonomic with a concrete case at hand.